crypto/cms/cms_env.c

```diff
@@ -48,6 +79,25 @@ static CMS_EnvelopedData *cms_enveloped_data_init(CMS_ContentInfo *cms)
     return cms_get0_enveloped(cms);
 }
 
+static CMS_AuthEnvelopedData *
+cms_auth_enveloped_data_init(CMS_ContentInfo *cms)
+{
+    if (cms->d.other == NULL) {
+        cms->d.authEnvelopedData = M_ASN1_new_of(CMS_AuthEnvelopedData);
+        if (!cms->d.authEnvelopedData) {
```

Comment:
nit : check against NULL:
`         if (cms->d.authEnvelopedData != NULL) { `

Reply:
It's actually `if (cms->d.authEnvelopedData == NULL) {`

Fixed now